CHANGELOG.rst

```diff
@@ -4,10 +4,11 @@ CHANGELOG
 This document describes changes between each past release.
 
 
-0.6.0 (unreleased)
+0.6.0 (2015-10-12)
 ==================
 
-- Nothing changed yet.
+- Supports requests-hawk 0.2.0 drawbacks. (#17)
```

Comment:
You need to be consistent. I would suggest to use the infinitive of the verb (support, add) everywhere.